README.yaml

```
@@ -67,7 +67,33 @@ description: |-
 # How to use this project
 usage: |-
   ```
-  TODO
+  module "atlantis" {
+    source = "git::https://github.com/cloudposse/terraform-aws-ecs-atlantis.git?ref=tags/0.1.0"
+
+    atlantis_enabled           = "true"
+    atlantis_gh_team_whitelist = "terraform:plan,engineering:*"
```

```suggestion
    atlantis_gh_team_whitelist = "operations:plan,engineering:*"
```

let's use a team name other than `terraform` since it's confusing and does not reflect the purpose of the team